docker-compose-base.yaml

```diff
@@ -174,7 +174,7 @@ services:
       - "-W"                                # wait roscore
       # - "-p"                                # people topic is simulation groundtruth
       - "-D"                                # detection
-      - "-a"                                # no resetrs individually
+      # - "-a"                                # no resetrs individually
```

Comment:
I remember that USB realsense should not be reset individually; we got issues if multiple containers reset USB simultaneously.

It may be better to configure this only for framos.